lib/purchases_flutter.dart

```diff
@@ -599,25 +599,26 @@ class Purchases {
     return result;
   }
 
-  /// iOS only. Use this function to retrieve the `PurchasesPaymentDiscount`
-  /// for a given `PurchasesPackage`.
+  /// iOS only.
   ///
-  /// Returns a [PaymentDiscount] object. Pass this object
+  /// Use this function to retrieve the `PromotionalOffer` to apply to a
```

Comment:
Why is `PromotionalOffer` here in \`backticks\` but elsewhere it's in `[square brackets]`?

Reply:
Oopsies